test/e2e/components/vl-map.js

```diff
@@ -1,6 +1,70 @@
 const { VlElement } = require('vl-ui-core').Test;
+const { By } = require('vl-ui-core').Test.Setup;
+const VlMapBaseLayer = require('./vl-map-baselayer');
+const VlMapSearch = require('./vl-map-search');
 
-class VlMap extends VlElement {  
+class VlMap extends VlElement {
+	async _getMap() {
+		return this.shadowRoot;
+	}
+
+	async getBaseLayers() {
+		const childElements = await this.findElements(By.css(this.selector + ' *'));
+		const tagNames = await Promise.all(childElements.map(element => element.getTagName()));
+		const baseLayerElements = childElements.filter((element, index) => tagNames[index].indexOf('vl-map-baselayer') > -1);
+		return Promise.all(baseLayerElements.map(async element => await new VlMapBaseLayer(this.driver, element)));
+	}
+	
+	async isEscapeKeyDisabled() {
+		return this.hasAttribute("disable-escape-key");
+	}
+
+	async isRotationDisabled() {
+		return this.hasAttribute("disable-rotation");
+	}
+	
+	async isMouseWheelZoomDisabled() {
+		return this.hasAttribute("disable-mouse-wheel-zoom");
+	}
+
+	async getOverviewMap() {
+		const map = await this._getMap();
+		await this.driver.wait(async () => {
+            const overviewMaps = await map.findElements(By.css('.ol-overviewmap'));
```

Comment:
lachend kakske

Reply:
Zie het niet direct in de code

Reply:
show whitespace characters aanzetten in je favoriete IDE, been there done that :-)

Reply:
Yep, sommige regels hebben tabs en sommige hebben spaces.

Reply:
we hebben echt dringend deftige code formatting rules nodig zodat ik gewoon kan autoformatten... @kevcraey mag ticket https://www.milieuinfo.be/jira/browse/UIG-292 niet wat hoger in backlog?

Reply:
In tussentijd kan je misschien gewoon de auto formatting van uw IDE gebruiken voor je commit? 
Op zich, zolang er geen code formatting rules zijn, maakt het niet uit of een file met tabs of met spaties is, zolang heel de file enkel het 1 of het ander heeft.

Reply:
Vergeet ook niet dat je bij reviewen kan aanzetten dat whitespace changes genegeerd worden.